dist/spec/index.html

```diff
@@ -26,6 +26,7 @@
  *   - .figure and .caption (HTML4) and figure and figcaption (HTML5)
```

Comment:
Can you revert the changes to this file?

Reply:
I think I have old version of bikeshed (or whatever it's called).